src/targets/chrome/docker.js

```diff
@@ -21,6 +21,50 @@ const getLocalIPAddress = () => {
   return ips[0];
 };
 
+// https://tuhrig.de/how-to-know-you-are-inside-a-docker-container/
+const getIsRunningInsideDocker = () =>
+  fs.existsSync('/proc/1/cgroup') &&
+  /docker/.test(fs.readFileSync('/proc/1/cgroup', 'utf8'));
+
+const getDockerContainerIPAddress = async dockerId => {
+  const { code, stdout } = await execa('docker', [
+    'inspect',
+    '-f {{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}',
+    dockerId,
+  ]);
+
+  if (code !== 0) {
+    throw new Error('Unable to determine IP of docker container');
+  }
+
+  return stdout;
+};
+
+const getHostExist = async hostname => {
+  try {
+    await execa('ping', ['-q', '-c 1', '-t 1', hostname]);
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+const getNetworkHost = async dockerId => {
+  if (getIsRunningInsideDocker()) {
+    // Gitlab's docker in docker service is exposed as "docker"
+    if (await getHostExist('docker')) {
```

Comment:
Humm, this is going to break my environment because we have a `docker.mycompany-network` server running, and `ping docker` resolves to that, even though I am running a local `docker` daemon that is what I actually want to talk to.

Reply:
I don't have a gitlab CI setup myself (but I will try to set one up) to better test/or propose an alternate solution. Let me set about trying to figure that out. (any way we can cut a release before this, since I think this is actually a change that will break us)

Reply:
I don’t really like this solution either, but didn’t really manage to find another solution. Thought about using `DOCKER_HOST`, but that’s not right either.

I’mma cut a release for you then.

Reply:
You can use my branch and setup if you want to save a few minutes :-)